client.js

```diff
@@ -60,14 +97,20 @@ const isValidURL = function (url) {
   return false;
 };
 
-//GOAL:If a message does not pass validation, it should be sent to the DLQ and removed from the input topic.
+const sendMsg = (message, token) => {
+  axios
+    .post(`${URL}/message`, { message }, { headers: { "X-Gotify-Key": token } })
+    .then(console.log("sendMsg", message))
+    .catch((err) => console.log("fn sendMsg", err.response));
```

Comment:
Log text needs to be more descriptive:
```suggestion
    .catch((err) => console.log("Error sending message:", err));
```

Reply:
Changed to:  console.log("Error sending message:", err));